src/decorators.ts

```diff
@@ -188,6 +189,13 @@ implements MethodRegistrationBase
   }
 }
 
+export interface ConfiguredClass<CT, T=unknown> {
+  init: InitConfigMethod;
+  cfgname: string;
+  arg: T;
+  ctor: CT;
+}
```

Comment:
Since this class is exposed to users (from `getConfiguredClass`), we need to clean up field names a bit.

Reply:
I agree, I don't like the current names.  What should they be called?

Reply:
I picked some longer names.